### DISCUSSION TITLE
Cant able to build verdaccio image with the source code

### DISCUSSION THREAD

**Discussion by kirankucharkanti** [verdaccio/verdaccio#2736] [Q&A]
Hi 

I downloaded source code from below link and trying to build the image locally but getting errors " ': No such file or directory". Can anyone plaese help me on this to resolve the issue.

https://github.com/verdaccio/verdaccio

**Comment by juanpicado** [comment]
hi @kirankucharkanti maybe you can elaborate a bit more. Be specific, steps etc etc 🙏🏼

**Comment by edclement** [comment]
Following up to what @juanpicado said, I would make sure you switch from the `master` branch to one of the [tags](https://github.com/verdaccio/verdaccio/tags) (e.g. `git checkout vX.X.X`). I've been able to successfully get both Verdaccio@5 (Latest stable release) and Verdaccio@6 running locally from source code without too much issue. I found some of the articles in the [wiki](https://github.com/verdaccio/verdaccio/wiki/Debugging-Verdaccio) helpful.

If you provide some examples of the steps you are taking and the exact error message(s) your are receiving it would be helpful in order for others to provide troubleshooting feedback. Based on you referencing the term _image_, are you trying to build a docker image? Information about your build environment (e.g. OS, docker version, etc) would be helpful. Verdaccio also publishes a pre-built image to the public dockerhub repo: https://hub.docker.com/r/verdaccio/verdaccio
